src/codegen/functions/parameter.py

```diff
@@ -11,11 +11,11 @@ def __init__(self, parameter_metadata):
         """Structure for storing function parameter metadata from scrapigen."""
         self._direction = parameter_metadata["direction"]
         self._parameters_name = camel_to_snake_case(parameter_metadata["name"])
-        self._type = parameter_metadata["type"]
-        self._ctypes_data_type = parameter_metadata["ctypes_data_type"]
-        self._python_data_type = parameter_metadata["python_data_type"]
-        self._description = parameter_metadata["python_description"]
-        self._python_type_annotation = parameter_metadata["python_type_annotation"]
+        self._type = parameter_metadata.get("type", None)
+        self._ctypes_data_type = parameter_metadata.get("ctypes_data_type", None)
+        self._python_data_type = parameter_metadata.get("python_data_type", None)
+        self._description = parameter_metadata.get("python_description", None)
+        self._python_type_annotation = parameter_metadata.get("python_type_annotation", None)
```

Comment:
FYI: https://docs.python.org/3/library/stdtypes.html#dict.get

> get(key[, default])
> Return the value for key if key is in the dictionary, else default. If default is not given, it defaults to None, so that this method never raises a [KeyError](https://docs.python.org/3/library/exceptions.html#KeyError).

Reply:
Understood. Removed setting default value as `None`.